projects/plugins/jetpack/extensions/index.json

```diff
@@ -93,5 +93,6 @@
 		"tock",
 		"videopress",
 		"wordads"
-	]
+	],
+	"_bundles": { "test": [ "blogging-prompt", "business-hours" ] }
```

Comment:
If we don't need to name bundles, this might as well be:
```
"_bundles": [
  [ "blogging-prompt", "business-hours" ], // bundled blocks
  "ai-chat" // single block
]
```